codebuild_specs/e2e_workflow_generated.yml

```diff
@@ -1925,19 +1949,28 @@ batch:
         image: $WINDOWS_IMAGE_2019
         variables:
           TEST_SUITE: src/__tests__/import_dynamodb_1.test.ts
-          CLI_REGION: ap-southeast-2
+          CLI_REGION: us-east-2
           USE_PARENT_ACCOUNT: true
       depend-on:
         - build_windows
         - upload_pkg_binaries
+    - identifier: aggregate_e2e_reports
```

Comment:
This is the only real change in this file.

Reply:
does this need regenerated? if we have a wait_for_all script

Reply:
yes, because the wait script needs the WAIT_FOR_IDS variable so that it knows which jobs to wait for.

Reply:
Oh i see, I mistakenly thought it was still using dependsOn all jobs but this is good